app/scripts/prunepackages.py

```diff
@@ -29,36 +27,77 @@
 ]
 
 REMOVE_GLOBS = [
+    # Only the Kolibri bundled C extensions for this build are needed. Keep the list of
+    # pruned ABIs and Python versions in sync with build.gradle.kts.
+    "common/kolibri/dist/cext/*/*/armv6l",
+    "common/kolibri/dist/cext/*/*/i686",
+    "common/kolibri/dist/cext/*/*/aarch64",
     "common/kolibri/dist/cext/cp27",
     "common/kolibri/dist/cext/cp36",
     "common/kolibri/dist/cext/cp37",
     "common/kolibri/dist/cext/cp38",
+    "common/kolibri/dist/cext/cp311",
```

Comment:
```suggestion
    # Keep "common/kolibri/dist/cext/cp39".
    # In app/build.gradle.kts, Chaquopy Python version is 3.9.
    "common/kolibri/dist/cext/cp311",
```

I think it would help to add a note here so we continue the thread from that comment you added in app/build.gradle.kts.